vms.tf

```diff
@@ -7,9 +7,11 @@ locals {
     : var.storage_type == "ha" ? module.netapp[0].netapp_endpoint : module.nfs[0].private_ip_address
   )
 
+  protocol_version = var.storage_type == "ha" && startswith(var.netapp_protocols[0], "NFS") ? split("v", var.netapp_protocols[0])[1] : "4.1"
```

Comment:
This should always be version 4.1 and have it active. That way the nfs server can support all pervious versions and 4.1 as well. No logic should be needed here.

Reply:
Agreed.

Reply:
After discussion with Thomas, we decided to keep this NFS version check logic in place.